src/tutorials/race_timer.md

```diff
@@ -0,0 +1,4383 @@
+
+![!Community Content Template](../img/RaceTimerTutorial/import_track.png){: .center loading="lazy" }
+
+Drag and drop the template **Race Timer Tutorial - Track** into the **Hierarchy** and **Deinstance** it.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/RaceTimerTutorial/cc_template_deinstance.mp4" type="video/mp4" />
+    </video>
+</div>
+
+If you enter **Play** mode you will notice the spawn position isn't in an ideal location for any player that joins the game. Move the **Spawn Point** so that it's before the starting position on the track.
+
+!!! tip
+    Pressing ++V++ will toggle the visibility of Gizmos. With visibility on, it's easy to see where to move objects such as the **Spawn Point**.
+
+![!Spawn Point](../img/RaceTimerTutorial/move_spawn_point.png){: .center loading="lazy" }
+
+## Starting Line
+
+When a race is about to start, we need to move the players to the starting line. Instead of grouping all players up in one spot, we can create a few positions along the starting line and randomly pick one to place the player at. There is still a possibility of players spawning in the same location, and to fix this it would require a more advanced system to keep track of which lanes have been assigned. For this tutorial we will keep it simple.
+
+### Creating Lane Positions
+
+!!! tip
+    As a visual aid to help move the starting positions, an object such as a **Cube** can be used. For performance it's recommended to use a **Group** when needing to reference a position for spawning in objects. **Group** objects have minimal properties (i.e. no **Rendering**) compared to other objects.
+
+1. Create a new group called **Starting Positions** inside the **Race Timer Tutorial - Track** group.
+2. Drag and drop a **Cube** from **Core Content** into the **Starting Positions** group.
+3. Set **Game Collision** to **Force Off** for the **Cube** object.
+4. Set **Camera Collision** to **Force Off** for the **Cube** object.
+5. Rename **Cube** to **Position**.
+6. Duplicate and space these out along the starting line.
+7. Set **Visibility** to **Force Off** for all **Position** objects.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/RaceTimerTutorial/create_start_positions.mp4" type="video/mp4" />
+    </video>
+</div>
+
+## Setup Script Hierarchy
+
+We need to setup the script folder and contexts in the **Hierarchy**.
+
+1. Create a folder called **Scripts** inside the **Race Timer Tutorial - Track** group.
+2. Create a **Client Context** inside the **Scripts** folder.
+3. Create a **Server Context** inside the **Scripts** folder.
+
+### Client Context vs Server Context
```

Comment:
Going by the outline it was recommended to touch on this subject since it's an advanced scripting tutorial.  So I kept it brief and linked to the networking page which I assume covers more of that subject in detail?  If it doesn't, we should definitely update it so that it contains best practices / example when handling stuff like purchases (not relating to Perks).